src/app/get-started/custom-workspace-tab/infrastructure-namespace-row/infrastructure-namespace-row.html

```diff
@@ -27,7 +27,7 @@
 
     <!-- Two and More Namespaces -->
     <che-pf-select-single config="ctrl.namespaceSelect.config" value="ctrl.namespaceSelect.value"
-      on-select="ctrl.namespaceSelect.onSelect($value)" ng-if="ctrl.namespacesNumber>1">
+      on-select="ctrl.namespaceSelect.onSelect($value)" ng-if="ctrl.namespacesNumber">
```

Comment:
The comment:  ```<!-- Two and More Namespaces -->```
Is it correct ```ng-if="ctrl.namespacesNumber>1" -> ng-if="ctrl.namespacesNumber"```?

Reply:
you're right, it was changed accidentally